mainutil/main.go

```diff
@@ -122,8 +122,8 @@ func Run(cmd *cobra.Command) {
 	}()
 	if err != nil && !errors.Is(err, context.Canceled) {
 		ctx := cmd.Context() // might be different from ctx set up in closure above
-		log := logutil.FromContext(ctx)
-		log.ErrorContext(ctx, "command failed", logutil.Err(err))
```

Comment:
It'd probably be better to have scopedError support in the slog Handler, so it works for all log calls.

Tangential: being able to use scopedError directly would also be good